frontend/src/scenes/settings/environment/SessionRecordingIngestionSettings.tsx

```diff
@@ -320,6 +322,32 @@ function UrlBlocklistOptions(): JSX.Element | null {
     )
 }
 
+function EventTriggerOptions(): JSX.Element | null {
+    const { eventTriggerConfig } = useValues(sessionReplayIngestionControlLogic)
+    const { updateEventTriggerConfig } = useActions(sessionReplayIngestionControlLogic)
+
+    return (
+        <div className="flex flex-col space-y-2 mt-4">
+            <div className="flex items-center gap-2 justify-between">
+                <LemonLabel className="text-base">User sends events</LemonLabel>
```

Comment:
```suggestion
                <LemonLabel className="text-base">Event emitted</LemonLabel>
```

maybe? or event sent 🤔 

not a massively strong opinion but they're not _really_ sent by users